CONTRIBUTING.md

```diff
@@ -1,52 +1,72 @@
 
 ## Enhancements
 
-If you have an idea or suggestion for an enhancement to the LabVIEW Icon Editor library, please use the [New Features](https://github.com/ni/labview-icon-editor/discussions/categories/new-features) discussion section. If you want to build it yourself, **please reach out to us before you dive into a new pull request.** The direction of the LabVIEW Icon Editor and which new features are added are discussed in our Discord Server and in [this GitHub discussions section](https://github.com/ni/labview-icon-editor/discussions/categories/new-features), and in the end, they are decided by the core team.
+If you have an idea or suggestion for an enhancement to the LabVIEW Icon Editor library, please use the [New Features](https://github.com/ni/labview-icon-editor/discussions/categories/new-features) discussion section. **please make sure to start a discussion about your changes.** The direction of the LabVIEW Icon Editor and which new features are added are discussed in our Discord Server and in [this GitHub discussions section](https://github.com/ni/labview-icon-editor/discussions/categories/new-features), and in the end, they are decided by the core team.
 
-Talking to us first about the enhancement you want to build will be the most likely way to get your pull request into the library (see Our Contributing Philosophy above). We would hate to see you write code you’re proud of, just to learn that we’ve already been working on the same thing, or that we feel doesn’t fit into the core library.
+Talking to us first via the discussions section about the enhancement you want to build will be the most likely way to get your pull request into the library (see Our Contributing Philosophy above). We would hate to see you write code you’re proud of, just to learn that we’ve already been working on the same thing, or that we feel doesn’t fit into the core library.
 
-### Contact us
+Once your idea has been selected for the next release of LabVIEW, a branch will be created that you can submit your pull request to.
 
-To ask about a possible enhancement, email us at {team_email_address}
+## Creating a discussion for a new feature
 
-### 💫 Pull Requests
+The first step to start an enhancement to the icon editor is to create a discussion on the [New Features](https://github.com/ni/labview-icon-editor/discussions/categories/new-features) discussion board. You must describe the problem that made you want to have this enhancement.
 
-If you want to contribute improvements or new features, we are happy to review your PR :)  
-Please use a meaningful commit message and add a little description of your changes.
-
-### Configuring your Development Environment
+### 💫 Pull Requests
 
-1. Install package dependencies `dragon install`
-2. {step_2}
-3. Test your changes by editing the icons of some VIs
-4. Run tests `caraya` and add new tests if you are introducing a new feature
+When submitting a PR, please follow these guidelines to ensure clarity and ease
```

Comment:
@svelderrainruiz It's possible to set up a template where a user can fill in the blanks almost like a form. @svelderrainruiz

Reply:
It should be doable using this
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository